components/Cat.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { StyleSheet, Text, View, TouchableOpacity, Image } from 'react-native';
+
+export const Cat = ({ cat, goToCat }) => {
+  return (
+    <TouchableOpacity style={styles.outerContainer} onPress={() => goToCat(cat)}>
+      <View style={styles.nameAndBreadBlock}>
+        <Text style={styles.catName}>{cat.name}</Text>
+        <Text style={styles.catBread}>{cat.bread}</Text>
+      </View>
+      <View style={styles.innerContainer}>
+        <Image source={cat.photo} style={styles.image} />
+        <Text style={styles.textInfo}>{cat.info}</Text>
+      </View>
+    </TouchableOpacity>
+  );
+};
+
+const styles = StyleSheet.create({
+  outerContainer: {
+    margin: 20,
+    padding: 10,
+    borderColor: '#C1807CFF',
+    borderWidth: 1,
+    borderRadius: 7,
+  },
+  innerContainer: {
```

Comment:
Could you please check the style's names? For example, 'imageContainer' instead of 'innerContainer' make it easy to understand the purpose of style